# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/roctracer/hotfix-no-tests.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./CMakeLists.txt.vanilla	2025-03-21 19:13:40.193292839 +0200
+++ ./CMakeLists.txt	2025-03-21 19:13:51.578107281 +0200
@@ -62,7 +62,7 @@
 
 ## Build tests
 if(${LIBRARY_TYPE} STREQUAL SHARED)
-  add_subdirectory(test)
+	#add_subdirectory(test)
 endif()
 
 ## Build Plugins
